WakaTime/WakaTime.swift

```diff
@@ -180,6 +180,8 @@ enum DeepLink: String {
 enum EntityType: String {
     case file
     case app
+    case domain
+    case url
```

Comment:
👋 @alanhamlett 

I wonder if I need to update WakaTime CLI for this? It seems to error out with 

```json
{
  "caller": "cmd/run.go:325",
  "func": "cmd.runCmd",
  "level": "error",
  "message": "failed to run command: sending heartbeat(s) failed: failed to load command parameters: failed to load heartbeat params: failed to parse entity type: invalid entity type \"url\"",
  "now": "2024-09-12T14:26:18+07:00",
  "os/arch": "darwin/arm64",
  "version": "v1.98.3"
}
```
when WakaTime.app tracks Safari.

Reply:
Oops, added with https://github.com/wakatime/wakatime-cli/pull/1090 Thanks!

Reply:
Thank you!